webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx

```diff
@@ -155,14 +156,14 @@ const ModelVersionHeader: React.FC<Props> = ({
   return (
     <header className={css.base}>
       <div className={css.headerContent}>
-        <div className={css.mainRow}>
-          <div className={css.title}>
+        <Row justifyContent="space-between" wrap>
+          <div className={css.mainRow}>
             <div className={css.versionBox}>V{modelVersion.version}</div>
-            <h1 className={css.versionName}>
-              {modelVersion.name ? modelVersion.name : `Version ${modelVersion.version}`}
-            </h1>
+            <Title truncate={{ tooltip: true }}>
```

Comment:
Do we wanna hide the version name?

### Before

<img width="907" alt="image" src="https://github.com/user-attachments/assets/002af89f-e235-4e25-a849-8da355b3cf46">


### After

<img width="824" alt="image" src="https://github.com/user-attachments/assets/76af2957-d2fc-4501-87cb-a2701213f0c7">

Reply:
No, good catch

Reply:
i dont see any changes.

Reply:
It's there now